tajo-pullserver/src/main/java/org/apache/tajo/pullserver/TajoPullServerService.java

```diff
@@ -403,12 +400,20 @@ public void decrementRemainFiles(FileRegion filePart, long fileStartTime) {
         minTime = fileSendTime;
       }
 
+      if (fileSendTime > 20 * 1000) {
+        LOG.warn("PullServer send too long time: filePos:" + filePart.position()
```

Comment:
I know you just moved this code. But, the original logging message needs to be corrected. Could you update the log message?